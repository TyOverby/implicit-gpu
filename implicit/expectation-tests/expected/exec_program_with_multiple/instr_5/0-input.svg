version https://git-lfs.github.com/spec/v1
oid sha256:40fe2548f61b913960669553d254f19d1c41e3cfce59bac604c8a53b64d6af5a
size 7847
