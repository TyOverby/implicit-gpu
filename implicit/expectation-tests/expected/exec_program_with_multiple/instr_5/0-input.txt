((4.0,4.0), (4.0,4.0))
((4.0,4.0), (4.0,4.0))
((4.0,4.0), (5.0,3.0))
((4.999999,5.0), (4.0,4.0))
((5.0,3.0), (5.0,3.0))
((5.0,3.0), (6.0,2.3430974))
((5.0,5.000001), (4.999999,5.0))
((6.0,2.3430974), (6.6617017,2.0))
((6.0,6.0), (5.0,5.000001))
((6.0,6.0), (6.0,6.0))
((6.6617017,2.0), (7.0,1.8359774))
((6.999999,7.0), (6.0,6.0))
((7.0,1.8359776), (8.0,1.4617925))
((7.0,7.000001), (6.999999,7.0))
((8.0,1.4617925), (9.0,1.2023916))
((8.0,8.0), (7.0,7.000001))
((8.0,8.0), (8.0,8.0))
((9.0,1.2023916), (10.0,1.0501516))
((9.0,9.0), (8.0,8.0))
((9.0,9.0), (9.0,9.0))
((10.0,1.0501516), (11.0,1.0))
((10.0,10.0), (9.0,9.0))
((10.0,10.0), (10.0,10.0))
((11.0,1.0), (11.0,1.0))
((11.0,1.0), (11.0,1.0))
((11.0,1.0), (12.0,1.0501516))
((11.0,11.0), (10.0,10.0))
((11.0,11.0), (11.0,11.0))
((12.0,1.0501516), (13.0,1.2023916))
((12.0,12.0), (11.0,11.0))
((12.0,12.0), (12.0,12.0))
((13.0,1.2023916), (14.0,1.4617925))
((13.0,13.0), (12.0,12.0))
((13.0,13.0), (13.0,13.0))
((14.0,1.4617925), (15.0,2.0))
((14.0,14.0), (13.0,13.0))
((14.0,14.0), (14.0,14.0))
((15.0,2.0), (15.0,2.0))
((15.0,2.0), (15.0,3.0))
((15.0,3.0), (15.0,4.0))
((15.0,4.0), (15.0,5.0))
((15.0,5.0), (15.0,6.0))
((15.0,6.0), (15.0,7.0))
((15.0,7.0), (15.0,8.0))
((15.0,8.0), (15.0,9.0))
((15.0,9.0), (15.0,10.0))
((15.0,10.0), (15.000001,11.0))
((15.0,12.0), (15.0,13.0))
((15.0,13.0), (15.0,14.0))
((15.0,14.0), (15.0,15.0))
((15.0,15.0), (14.0,14.0))
((15.0,15.0), (15.0,15.0))
((15.0,15.0), (15.0,15.0))
((15.000001,11.0), (15.0,12.0))
